src/core/server/config/__tests__/config_service.test.ts

```diff
@@ -18,8 +18,13 @@
  */
 
 /* tslint:disable max-classes-per-file */
+
```

Comment:
Oh, no, I realized it will be a rebase nightmare for me since I have 3 more PRs based on this one that change these tests :/ Would you mind I prepare a dedicated PR to get rid of all __tests__ in `src/core/server` right after https://github.com/elastic/kibana/pull/22190?

Reply:
works for me!